content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -76,71 +76,92 @@ Here we suggest some specific seed scripts for different situations. You are fre
 <tab>
 
 1. Create a new file named `seed.ts`. This can be placed anywhere within your projects folder structure. The below example places it in the `/prisma` folder.
-2. In the `seed.ts` file, import Prisma Client, initialize it and create some records.
-
-  <details><summary>Expand to view example seed script</summary>
-
-    ```js
-    import { PrismaClient } from '@prisma/client'
-    const prisma = new PrismaClient()
-
-    async function main() {
-      const alice = await prisma.user.upsert({
-        where: { email: 'alice@prisma.io' },
-        update: {},
-        create: {
-          email: 'alice@prisma.io',
-          name: 'Alice',
-          posts: {
-            create: {
-              title: 'Check out Prisma with Next.js',
-              content: 'https://www.prisma.io/nextjs',
-              published: true,
-            },
-          },
-        },
-      })
-
-      const bob = await prisma.user.upsert({
-        where: { email: 'bob@prisma.io' },
-        update: {},
-        create: {
-          email: 'bob@prisma.io',
-          name: 'Bob',
-          posts: {
-            create: [
-              {
-                title: 'Follow Prisma on Twitter',
-                content: 'https://twitter.com/prisma',
-                published: true,
-              },
-              {
-                title: 'Follow Nexus on Twitter',
-                content: 'https://twitter.com/nexusgql',
-                published: true,
-              },
-            ],
-          },
-        },
-      })
-      console.log({ alice, bob })
-    }
-
-    main()
-      .then(async () => {
-        await prisma.$disconnect()
-      })
-      .catch(async (e) => {
-        console.error(e)
-        await prisma.$disconnect()
-        process.exit(1)
-      })
-    ```
-
-  </details>
+2. In the `seed.ts` file, import Prisma Client, initialize it and create some records. As an example, take the following Prisma schema with a `User` and `Post` model:
 
-3. Add `typescript`, `ts-node` and `@types/node` development dependencies:
+   ```prisma file=schema.prisma
+   generator client {
+     provider = "prisma-client-js"
+   }
+   
+   datasource db {
+     provider = "postgresql"
+     url      = env("DATABASE_URL")
+   }
+   
```

Comment:
If you want to keep the schema shorter, you could leave this off I think.

Reply:
Thanks, removed